package.json

```diff
@@ -422,118 +422,118 @@
                     "group": "MS_SQL@1"
                 },
                 {
-                    "command": "mssql.filterNode",
-                    "when": "view == objectExplorer && config.mssql.enableRichExperiences && viewItem =~ /\\bfilterable=true\\b.*\\bhasFilters=false\\b/",
-                    "group": "inline@1"
+                    "command": "mssql.scriptSelect",
+                    "when": "view == objectExplorer && viewItem =~  /\\btype=(Table|View)\\b/",
+                    "group": "MS_SQL@1"
```

Comment:
Instead of dumping everything into a generic "MSSQL" group, perhaps we should group them by function?

`1_MSSQL_serverAndDb@N`
`2_MSSQL_script@N`
`3_MSSQL_object@N`